libs/designsystem/slide/src/slides.component.scss

```diff
@@ -97,4 +97,10 @@ $bullet-inactive-opacity: 0.2;
   swiper-slide:has([slideStretchHeight]) {
     height: auto;
   }
+
+  .kirby-text-medium {
+    @include utils.slotted(span, h1, h2, h3, h4, h5, h6) {
```

Comment:
Dont think span is needed, as we only project h1-h6.
Consider adding a 'title' class instead of styling on the kirby-text-medium utility class.